Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp

```diff
@@ -134,6 +142,18 @@ namespace VehicleDynamics
                 continue;
             }
 
+            if (maxSpeedImpulse > 0.0)
+            {
+                if (pidCommand > maxSpeedImpulse)
+                {
```

Comment:
better:

if (maxSpeedImpulse > 0.0) 
{ // 0.0 means no limit will be applied
    pidCommand = AZStd::clamp(pidCommand, -maxSpeedImpulse, maxSpeedImpulse);
}

Reply:
(just apply formatting to the above